README.md

```diff
@@ -70,15 +72,15 @@ kubectl -n keptn get pods -l run=keptn-service-template-go
 Adapt and use the following command in case you want to up- or downgrade your installed version (specified by the `$VERSION` placeholder):
 
 ```console
-kubectl -n keptn set image deployment/keptn-service-template-go keptn-service-template-go=keptnsandbox/keptn-service-template-go:$VERSION --record
+helm upgrade -n keptn --set keptnservice.image.tag=$VERSION keptn-service-template-go chart/
```

Comment:
Should this be `keptnservice.image.tag` or just `image.tag`?

Reply:
`image.tag` is the correct key :)